tests/integration/test_simplified_workflows.py

```
@@ -0,0 +1,556 @@
+                if 'labeler' in step.get('uses', '').lower():
+                    # Labeler step should not have conditional if based on config check
+                    assert 'check-config' not in str(step.get('if', ''))
+    
+    def test_label_uses_labeler_action(self, label_workflow):
+        """Verify workflow uses the labeler action."""
+        workflow_content = Path('.github/workflows/label.yml').read_text()
+        assert 'actions/labeler' in workflow_content
+    
+    def test_label_has_checkout_step(self, label_workflow):
+        """Verify workflow checks out code."""
+        workflow_content = Path('.github/workflows/label.yml').read_text()
```

Ruff (0.14.6)

Local variable `workflow_content` is assigned to but never used

Remove assignment to unused variable `workflow_content`

(F841)